src/App.jsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { BrowserRouter, Switch, Route, Redirect } from 'react-router-dom';
```

Comment:
This is the import that will have to go

```suggestion
import { BrowserRouter, Switch, Route } from 'react-router-dom';
```